src/mainboard/clevo/tgl-u/variants/nv40mz/romstage.c

```diff
@@ -19,3 +19,5 @@ void variant_configure_fspm(FSPM_UPD *memupd)
 
 	memcfg_init(memupd, &board_cfg, &spd_info, half_populated);
 }
+
+
```

Comment:
Please remove, extra new liens added

Reply:
fixed in 31f28991bb1c1c1425e8a81e98819326d217a534